docs/docs-beta/docs/guides/external-systems/apis.md

```diff
@@ -1,4 +1,53 @@
 ---
 title: Connecting to APIs
 sidebar_position: 20
----
\ No newline at end of file
+---
+
+This guide describes how to connect to and interact with APIs in dagster.
```

Comment:
I think we could 1-3 sentences motivating the use of Resources: what they are and why we use them.

Reply:
+1